### PR TITLE
Do not use '_' in unit tests

### PR DESCRIPTION
**Pull request by guedou**
This PR removes '_' in unit tests. I add some issues on Python 3.6 on osx.

### CODE REVIEW

File: scapy/contrib/diameter.uts [hunk 1]
```diff
@@ -209,8 +209,10 @@ raw(ans1) == b'\x01\x00\x00\x14\x00\x00\x01\x01\x00\x00\x00\x00\x00\x00\x04\xd2\
 = Same answer identified by its code or abbreviation
 ans1b = DiamAns (257, drHbHId=1234, drEtEId=5678)
 ans1c = DiamAns ('CEA', drHbHId=1234, drEtEId=5678)
-raw(ans1b) == raw(ans1), raw(ans1c) == raw(ans1)
-_ == (True, True)
+a = raw(ans1b) == raw(ans1)
+b = raw(ans1c) == raw(ans1)
+a, b
+(a, b) == (True, True)
```
Comment (p-l-) [inline scapy/contrib/diameter.uts]: Just `a and b` would do.

File: scapy/contrib/homeplugav.uts [hunk 2]
```diff
@@ -48,39 +48,44 @@ HomePlugAV()/SnifferIndicate()
 pkt = HomePlugAV()/SetEncryptionKeyRequest()
 pkt.NMK = "A" * 16
 pkt.DAK = "B" * 16
-raw(pkt)
-_ == b"\x00P\xa0\x00\xb0R\x00AAAAAAAAAAAAAAAA\x00\xff\xff\xff\xff\xff\xffBBBBBBBBBBBBBBBB"
+r = raw(pkt)
+r
+r == b"\x00P\xa0\x00\xb0R\x00AAAAAAAAAAAAAAAA\x00\xff\xff\xff\xff\xff\xffBBBBBBBBBBBBBBBB"
```
Comment (p-l-) [inline scapy/contrib/homeplugav.uts]: Don't you need `assert` here?

File: scapy/contrib/homeplugav.uts [hunk 3]
```diff
@@ -48,39 +48,44 @@ HomePlugAV()/SnifferIndicate()
...
 
 pkt = HomePlugAV()/ReadMACMemoryRequest()
 pkt.Address = 0x31337
 pkt.Length = 0x666
-raw(pkt)
-_ == b"\x00\x08\xa0\x00\xb0R7\x13\x03\x00f\x06\x00\x00"
+r = raw(pkt)
+r
+r == b"\x00\x08\xa0\x00\xb0R7\x13\x03\x00f\x06\x00\x00"
```
Comment (p-l-) [inline scapy/contrib/homeplugav.uts]: Same.

File: scapy/contrib/homeplugav.uts [hunk 4]
```diff
@@ -48,39 +48,44 @@ HomePlugAV()/SnifferIndicate()
...
+r
+assert(r == b"\x00$\xa0\x00\xb0R\x02\x00f\x067\x13\x00\x00")
 
 pkt = HomePlugAV()/SnifferRequest()
 pkt.SnifferControl = 0x1
-raw(pkt)
-_ == b"\x004\xa0\x00\xb0R\x01"
+r = raw(pkt)
+r
+r == b"\x004\xa0\x00\xb0R\x01"
```
Comment (p-l-) [inline scapy/contrib/homeplugav.uts]: Same

File: scapy/contrib/homeplugav.uts [hunk 5]
```diff
@@ -48,39 +48,44 @@ HomePlugAV()/SnifferIndicate()
...
 
 pkt = HomePlugAV(_xstr)
-ReadModuleDataConfirmation in pkt
-_ == True
-(pkt[ReadModuleDataConfirmation].ModuleID == 2, pkt[ReadModuleDataConfirmation].checksum == 4177890656, pkt[ReadModuleDataConfirmation].DataLen == 1024, pkt[ReadModuleDataConfirmation].Offset == 0)
-_ == (True, True, True, True)
+assert ReadModuleDataConfirmation in pkt
+r = (pkt[ReadModuleDataConfirmation].ModuleID == 2, pkt[ReadModuleDataConfirmation].checksum == 4177890656, pkt[ReadModuleDataConfirmation].DataLen == 1024, pkt[ReadModuleDataConfirmation].Offset == 0)
```
Comment (p-l-) [inline scapy/contrib/homeplugav.uts]:
```python
assert pkt[ReadModuleDataConfirmation].ModuleID == 2
assert pkt[ReadModuleDataConfirmation].checksum == 4177890656
assert pkt[ReadModuleDataConfirmation].DataLen == 1024
assert pkt[ReadModuleDataConfirmation].Offset == 0
```

File: scapy/contrib/homeplugav.uts [hunk 6]
```diff
@@ -48,39 +48,44 @@ HomePlugAV()/SnifferIndicate()
...
+r 
+assert r == (True, True, True, True)
 
-ModulePIB(pkt.ModuleData, pkt.Offset, pkt.DataLen)
-(_.NMK == b"z]\xa9\xe2]\xedR\x8b\x85\\\xdf\xe8~\xe9\xb2\x14", _.DAK == b"\x1f\xedHu\x85\x16>\x86\x1aKM\xd2\xe91\xfc6")
-_ == (True, True)
+p = ModulePIB(pkt.ModuleData, pkt.Offset, pkt.DataLen)
+r = (p.NMK == b"z]\xa9\xe2]\xedR\x8b\x85\\\xdf\xe8~\xe9\xb2\x14", p.DAK == b"\x1f\xedHu\x85\x16>\x86\x1aKM\xd2\xe91\xfc6")
```
Comment (p-l-) [inline scapy/contrib/homeplugav.uts]:
```python
assert p.NMK == b"z]\xa9\xe2]\xedR\x8b\x85\\\xdf\xe8~\xe9\xb2\x14"
assert p.DAK == b"\x1f\xedHu\x85\x16>\x86\x1aKM\xd2\xe91\xfc6"
```

File: scapy/contrib/homeplugav.uts [hunk 7]
```diff
@@ -115,5 +120,7 @@ string = b"goodchoucroute\x00\x00"
 pkt = WriteModuleDataRequest(ModuleData=string)
 pkt = WriteModuleDataRequest(pkt.build())
 pkt.show()
-(pkt.checksum == chksum32(pkt.ModuleData), pkt.DataLen == len(pkt.ModuleData))
-_ == (True, True)
+a = pkt.checksum == chksum32(pkt.ModuleData)
+b = pkt.DataLen == len(pkt.ModuleData)
+a, b
```
Comment (p-l-) [inline scapy/contrib/homeplugav.uts]:
```python
a and b
```

File: test/fields.uts [hunk 8]
```diff
@@ -27,12 +27,15 @@ assert LELongField("test", None).addfield(None, b"", 0x44434241) == b'ABCD\x00\x
...
 m = MACField("foo", None)
-m.i2m(None, None)
-assert( _ == b"\x00\x00\x00\x00\x00\x00" )
-m.getfield(None, b"\xc0\x01\xbe\xef\xba\xbeABCD")
-assert( _ == (b"ABCD","c0:01:be:ef:ba:be") )
-m.addfield(None, b"FOO", "c0:01:be:ef:ba:be")
-assert( _ == b"FOO\xc0\x01\xbe\xef\xba\xbe" )
+r = m.i2m(None, None)
+r
+assert(r == b"\x00\x00\x00\x00\x00\x00" )
```
Comment (p-l-) [inline test/fields.uts]: You don't nee parenthesis in `assert` statements (you have a lot of those).

File: test/regression.uts [hunk 9]
```diff
@@ -565,8 +571,8 @@ x=Ether()/IP(version=4)/UDP()
 y=Ether()/IP()/UDP(dport=4)
 z=Ether()/IP()/UDP()/NTP()
 t=Ether()/IP()/TCP()
-x==y, x==z, x==t, y==z, y==t, z==t, w==x
-_ == (False, False, False, False, False, False, True)
+r = (x==y, x==z, x==t, y==z, y==t, z==t, w==x)
```
Comment (p-l-) [inline test/regression.uts]:
```python
x != y and x != z and x != t and y != z and y != t and z != t and w == x
```

File: scapy/contrib/diameter.uts (hunk 1 above)
Comment (guedou) [reply scapy/contrib/diameter.uts]: Done

File: scapy/contrib/homeplugav.uts (hunk 2 above)
Comment (guedou) [reply scapy/contrib/homeplugav.uts]: Good catch !

File: scapy/contrib/homeplugav.uts (hunk 3 above)
Comment (guedou) [reply scapy/contrib/homeplugav.uts]: Done

File: scapy/contrib/homeplugav.uts (hunk 4 above)
Comment (guedou) [reply scapy/contrib/homeplugav.uts]: Done

File: scapy/contrib/homeplugav.uts (hunk 5 above)
Comment (guedou) [reply scapy/contrib/homeplugav.uts]: Done

File: scapy/contrib/homeplugav.uts (hunk 6 above)
Comment (guedou) [reply scapy/contrib/homeplugav.uts]: Done

File: scapy/contrib/homeplugav.uts (hunk 7 above)
Comment (guedou) [reply scapy/contrib/homeplugav.uts]: Done

File: test/fields.uts (hunk 8 above)
Comment (guedou) [reply test/fields.uts]: Done, and the other ones `assert`.

**Comment by guedou** [comment]
@p-l- done. That was not fun at all =)

**Comment by p-l-** [comment]
I don't know why the Windows tests have not run on this one...

**Comment by p-l-** [comment]
I've launched them manually.

**Pull request merged**